### ISSUE TITLE
Home-Info Mode gives blank page

### ISSUE THREAD

**Issue by everetr** [adityatelange/hugo-PaperMod#227]
I tried the "master" branch as of now, v3.0, and v4.0. My Hugo version is "v0.80.0/extended linux/amd64 BuildDate: unknown". Manjaro Linux. **Edit:** I'm having the same issue with Regular Mode. Only Profile Mode works as expected. Same issue on Windows. Same with Hugo 0.74.0.

`config.yml`:

```yml
theme: PaperMod
params:
    homeInfoParams:
        Title: Hi there wave
        Content: Can be Info, links, about... 
```

Then I run `hugo server -D` but all I ever see is a blank page:

![image](https://user-images.githubusercontent.com/9023041/107453848-bfbb2700-6b19-11eb-89c3-e5bf1a1b2cae.png)

**Comment by everetr** [comment]
I'm having the same issue with Regular Mode. Only Profile Mode works as expected.

**Comment by adityatelange** [comment]
> I'm having the same issue with Regular Mode. Only Profile Mode works as expected.

Yes this is a buggy theme, please help me make it better

**Comment by everetr** [comment]
I figured it out. This issue occurs when the site has no posts. I was starting a site from scratch with no example posts, so the `content/post/` directory was empty.

It makes sense that Regular Mode would show up blank without posts. It's less obvious that Home-Info mode would be completely blank--even missing Title, Content, and Social Icons--without posts, but now I know. I'll close this issue.

**Comment by adityatelange** [comment]
>  It's less obvious that Home-Info mode would be completely blank--even missing Title, Content, and Social Icons

True. I didn't knew that was the case, I searched for this and found that there was a case where home-info contents were shown even on paginated pages, 
523098f87e9abe68ad8f217844fe62432840519c, that is the reason a post is needed to display home-info contents.

I'll see if we can find a workaround for this

**Comment by adityatelange** [comment]
@everetr This should be fixed by 143eb18

Let me know if that doesn't work

**Comment by everetr** [comment]
It works. Thank you for that.

Now, even when there are no posts, a Home-Info site still shows the Title, Content, and Social Icons.

**Comment by jehoshua7** [comment]
I installed this theme today. Initially after trying

`$ hugo server --source=exampleSite --themesDir=../..
Error: open /~/Hugo/mysite/themes/PaperMod/exampleSite: no such file or directory`

I realised there was no exampleSite directory. Then on simply using

`hugo server`

There was a blank page.I assume this is not a bug and simply indicates there is no local content (yet). Is that correct ? The page does show the open moon icon (Alt + T) up the top left hand corner, and " © 2021 Powered by Hugo & PaperMod " down the bottom.
